packages/framework/src/Model/SalesRepresentative/SalesRepresentativeDataFactoryInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\SalesRepresentative;
+
+interface SalesRepresentativeDataFactoryInterface
```

Comment:
please remove this interface, we decided not to use them anymore

Reply:
removed